utilities/poap/puppet_agent_install.py

```diff
@@ -0,0 +1,239 @@
+#!/isan/bin/python
+#md5sum=ec7497f2087cd87b8bccb06eb1ae61f8
+# The line above is the embedded md5sum of this file taken without this line.
+# The md5sum is an added level of script integrity verification beyond the
+# basic tftp ip checksum. The script will also perform an integrity check
+# on the image and configuration files by downloading corresponding files
+# with .md5 extensions. The md5sum value can be generated for this file by
+# issuing the following command:
+#  f=puppet_agent_install.py ; cat $f | sed '/^#md5sum/d' > $f.md5 ; sed -i "s/^#md5sum=.*/#md5sum=$(md5sum $f.md5 | sed 's/ .*//')/" $f
+#
+
+import os
+from datetime import datetime
+import subprocess
+import shlex
+import sys
+
+###############################################################################
+# START OF USER-CONFIGURABLE PARAMETERS
+# These user-configurable parameters should be updated to reflect the local
+# automation environment.
+###############################################################################
+
+# (Required) RPM_NAME = The puppet agent release RPM
+# **************************************************************
+# *** IMPORTANT: CHOOSE THE RIGHT RPM FOR YOUR ENVIRONMENT! ****
+# **************************************************************
+bash_shell_rpm = 'puppetlabs-release-pc1-cisco-wrlinux-5.noarch.rpm'
+guestshell_rpm = 'puppetlabs-release-pc1-el-7.noarch.rpm'
+RPM_NAME = bash_shell_rpm
+
+
+# (Required) RPM_URI = The download URI for the RPM
+# RPM_URI = 'ftp://1.2.3.4/'
+RPM_URI = 'http://yum.puppetlabs.com/'
+
+# (required) PUPPET_SERVER = The DNS name or IP address of the agent's puppet server
```

Comment:
I think we should remove cisco specific information in these USER-CONFIGURABLE PARAMETERS and replace them with generic sample data.